numpy/core/fromnumeric.py

```diff
@@ -890,6 +890,13 @@ def argsort(a, axis=-1, kind='quicksort', order=None):
     array([[0, 1],
            [0, 1]])
 
+    >>> np.unravel_index(np.argsort(x, axis=None), x.shape)
+    (array([0, 1, 1, 0]), array([0, 0, 1, 1]))
+    >>> x[np.unravel_index(np.argsort(x, axis=None), x.shape)]
```

Comment:
I'm not sure about it.
You are write that only the first line give indication on how argsort works.
But do the average numpy user know about advanced indexing and argument packing?
I feel like retrieving the list of indices that sort the array elements is the most basic thing we want argsort to do.

Maybe removing the second line but keeping the last one is a good option since even without knowing advanced indexing anyone can make a loop from the last line and retrieve the second line.